code/modules/power/singularity/singularity.dm

```diff
@@ -222,10 +220,11 @@
 			allowed_size = STAGE_THREE
 		if(1000 to 1999)
 			allowed_size = STAGE_FOUR
-		if(2000 to 2999)
+		if(2000 to INFINITY)
 			allowed_size = STAGE_FIVE
 		if(3000 to INFINITY)

```

Comment:
This case will never be reached, since the previous case is a complete superset of it.